Patch by Robert Scheck <robert@fedoraproject.org> for popt <= 1.16 which changes
$(pkgconfigdir) on 64 bit systems from /usr/lib/pkgconfig to /usr/lib64/pkgconfig.
Using $(libdir)/pkgconfig rather $(prefix)/lib/pkgconfig seems to be common when
searching on the Internet. This patch however is not really compliant with Fedora
/%{_lib} vs. %{_libdir} handling before the UsrMove with Fedora 17. Alternatively
--with-pkgconfigdir or similar should be implemented.

This patch was proposed to upstream: http://rpm5.org/community/popt-devel/0265.html

--- popt-1.16/Makefile.in			2010-05-04 22:55:59.000000000 +0200
+++ popt-1.16/Makefile.in.pkgconfig		2013-11-24 15:06:43.000000000 +0100
@@ -370,7 +370,7 @@
 libpopt_la_SOURCES = popt.c poptparse.c poptconfig.c popthelp.c poptint.c
 libpopt_la_LDFLAGS = -no-undefined @LTLIBINTL@ @LTLIBICONV@ \
 	$(am__append_1)
-pkgconfigdir = $(prefix)/lib/pkgconfig
+pkgconfigdir = $(libdir)/pkgconfig
 pkgconfig_DATA = popt.pc
 man_MANS = popt.3
 BUILT_SOURCES = popt.pc # popt.lcd
--- popt-1.16/Makefile.am			2010-05-04 22:55:54.000000000 +0200
+++ popt-1.16/Makefile.am.pkgconfig		2013-11-24 15:02:21.000000000 +0100
@@ -47,7 +47,7 @@
 libpopt_la_SOURCES = popt.c poptparse.c poptconfig.c popthelp.c poptint.c
 libpopt_la_LDFLAGS = -no-undefined @LTLIBINTL@ @LTLIBICONV@
 
-pkgconfigdir = $(prefix)/lib/pkgconfig
+pkgconfigdir = $(libdir)/pkgconfig
 pkgconfig_DATA = popt.pc
 
 if HAVE_LD_VERSION_SCRIPT
